css/login.css

```diff
@@ -0,0 +1,84 @@
+body{
+    box-sizing: border-box;
+    position: relative;
+}
+
+#external {
+    background-image:"../assets/feedback-banner.jpg";
+    height: 35rem;
+    position:relative;
+}
+
+/*@Alex I forgot to remove the url
+I cannot see from the code above it, but you asked about the relative and absolute so i hope it was about the footer/body relation.
+In order to position the footer relatively to the body (because i want to always stay at the bottom of the page so at ;the bottom of the body), I gave it position absolute, which is positioned after the first non-static ancestor element. In this case, the body which has to be relative. And because the properties are :
+ right: 0;
+    bottom: 0;
+    left: 0;
+
+it will always tend towards the left right extremities and as to the bottom as it can.
+*/
+
+#login {
+    position: absolute;
+    margin-left: 3rem;
+    margin-top: 3rem;
+}
+
+div.feedback {
+    width: 100%;
+}
+
+.floated {
+    float: left;
+}
+
+img.floated {
+    width: 25%;
+    margin-left: 12.5%;
+    margin-right: 12.5%;
+}
+p.floated {
+    width: 35%;
+}
+
+h1.floated {
+    width: 50%;
+    margin-bottom: 5rem;
+    margin-top: 5rem
+}
+
+.clearfix {
+    clear:both;
+}
+
+.floated2 {
+    float: right;
+}
+
+img.floated2 {
```

Comment:
I'm going to leave the extra specificity aside (already asked about that previously), and inquire whether we're also going to have a `.floated33&1/3`. :P I, for one, tend to stay well clear of numeric indices in selectors. For starters, because you usually have better options if you're targeting a particular child, but also because I like my names to be more suggestive. In other words, how could we rename this class to best describe its purpose, its intended usage?